anthos-bm-apigee/docs/quickstart.md

```diff
@@ -0,0 +1,590 @@
+Checking Apigee Containers ...
+Checking Apigee Containers ...
+Checking Apigee Containers ...
+Apigee is Ready
+Anthos on bare metal installation complete!
+Run [export KUBECONFIG=/home/tfadmin/bmctl-workspace/apigee-cluster/apigee-cluster-kubeconfig] to set the kubeconfig
+Run the [/home/tfadmin/login.sh] script to generate a token that you can use to login to the cluster from the Google Cloud Console
+```
+
+---
+### Verify and interacting with the Baremetal cluster
+
+You can find your cluster's `kubeconfig` file on the admin machine in the `bmctl-workspace` directory. To verify your deployment, complete the following steps
+
+1. SSH into the admin host _(if you are not already inside it)_:
+```sh
+# You can copy the command from the output of Terraform run from the previous step
+gcloud compute ssh tfadmin@apigee-cluster-abm-ws0-001 --project=<YOUR_PROJECT> --zone=<YOUR_ZONE>
+```
+
+2. Set the `KUBECONFIG` environment variable with the path to the cluster's configuration file to run `kubectl` commands on the cluster.
+```sh
+export CLUSTER_ID=apigee-cluster
+export KUBECONFIG=$HOME/bmctl-workspace/$CLUSTER_ID/$CLUSTER_ID-kubeconfig
+kubectl get nodes
+```
+
+You should see the nodes of the cluster printed, _similar_ to the output below:
+```sh
+NAME          STATUS   ROLES    AGE   VERSION
+apigee-cluster-abm-cp1-001   Ready    master   17m   v1.18.6-gke.6600
+apigee-cluster-abm-w1-001    Ready    <none>   14m   v1.18.6-gke.6600
+apigee-cluster-abm-w2-001    Ready    <none>   14m   v1.18.6-gke.6600
+apigee-cluster-abm-w3-001    Ready    <none>   14m   v1.18.6-gke.6600
+```
+
+
+#### Interacting with the cluster via the GCP console
+
+During the setup process, your cluster will be auto-registered in Google Cloud using [Connect](https://cloud.google.com/anthos/multicluster-management/connect/overview). In order to interact with the cluster from the GCP console you must first ***login*** to the cluster.
+
+The [Logging into the Anthos bare metal cluster](login.md) explains how you can do it.
+
+---
+
+
+### Cleanup
+
+You can cleanup the cluster setup in two ways:
+
+#### 1. Using Terraform
+
+- First deregister the cluster before deleting all the resources created by Terraform
+  ```sh
+  # SSH into the admin host
+  gcloud compute ssh tfadmin@apigee-cluster-abm-ws0-001 --project=<YOUR_PROJECT> --zone=<YOUR_ZONE>
+
+  # Reset the cluster
+  export CLUSTER_ID=apigee-cluster
+  export KUBECONFIG=$HOME/bmctl-workspace/$CLUSTER_ID/$CLUSTER_ID-kubeconfig
+  sudo bmctl reset --cluster $CLUSTER_ID
+
+  # logout of the admin host
+  exit
+  ```
+
+- Then, use Terraform to delete all resources.
+  ```sh
+  # to be run from the root directory of this repo
+  terraform destroy --auto-approve
+  ```
+ - Deregister cluster from the Cloud hub membership.
+
+#### 2. Delete the entire Google Cloud project
+- Directly [delete the project](https://console.cloud.google.com/cloud-resource-manager) from the console
+
+#### 3. Delete Apigee Organization
```

Comment:
I think the steps 3 and 4 are not different ways of `cleanup`. The first line under this section says: `You can cleanup the cluster setup in two ways:`. So I will make sure that the two ways are nested under one bullet and these two news ones are added as extra cleanup steps:

---
#### 1. Cleanup resources. You can cleanup the cluster setup in two ways:
    
    - Using Terraform
    - Delete the entire Google Cloud project

#### 2. Delete Apigee Organization
#### 3. Clean Temporary files

Reply:
Fixed